mainapp/templates/account.html

```diff
@@ -15,7 +21,46 @@ <h1>Account</h1>
                 <li class="list-group-item">User type: {{ user.user_type_string }}</li>
             </ul>
         </div>
+        <div class="col">
+        {% if user.review_set %}
+            <h1> Submitted Review: </h1>
+
+            <div class="mb-3 text-center location-results">
+                {% for review in user.review_set.all %}
+                <div class="card mb-4 shadow-sm "style="max-width: 27rem;">
+                    <div class="card-header">
+                        <ul class="list-unstyled">
+                            <a href="/location/{{review.location.id}}/">{{ review.location.address }}</a>
```

Comment:
I think this way is safer and django-ish:
```
<a href="{% url 'location' location.id %}">
```

Reply:
@alldne updated